java/runtime/src/main/java/org/ray/runtime/runner/worker/DefaultWorker.java

```diff
@@ -15,6 +16,7 @@
   public static void main(String[] args) {
     try {
       System.setProperty("ray.worker.mode", "WORKER");
+      Thread.setDefaultUncaughtExceptionHandler(new WorkerUncaughtExceptionHandler());
```

Comment:
Maybe just use a lambda here? As the handler is pretty simple.

Reply:
This is not a callback function, can this be a lambda?

Reply:
Yes, all interfaces with `@FunctionalInterface` can be represented as a lambda